examples/bigquery-table-access-pattern-analysis/README.md

```diff
@@ -17,31 +17,40 @@ with its destination table set to return_purchases table. The source table of th
+This tool helps identify pipeline optimisation points, by pinpointing tables with high difference of writing and reading frequency throughout the data warehouse queries. High discrepancy between the write and read frequency of a table might be a good starting point for identifying optimisation points. Using this tool, we will also be able to visualise the pipelines that are involved with a table of interest, you can then further analyse the pipeline type and its scheduling pattern, and also each of the job of each pipelines, and pinpoint problems or optimisations from there. 
 
 ![](assets/pipeline-example.gif)
 
 #### <b>Analysing the Result</b>
 As can be seen from the GIF, the tool will visualise all the pipelines associated with a table. To be specific, this includes all query jobs that has this table of interest as its source or destination table. As mentioned above, for every query job, there are source table(s) and also a single destination table. 
 
-The tables that are involved in the pipelines associated with a table employs the below logic:
+The tables that are involved in the pipelines associated with a table (all tables that are included when the pipeline graph of a table of interest is plotted) employs the below logic:
 
 For every query jobs that has the table of interest as one of its source tables or destination table, 
 * For every source table(s) of every query job that has the table of interest as one of its source table(s), recursively find query jobs that has this source table as its destination table, and get its source table(s). 
 * For every destination table of every query job that has the table of interest as its destination table, recursively find query jobs that has destination table as its source table, and get its destination table.
 
-As seen from the GIF too, for every tables that are involved in the pipeline of the table of interest, you can toggle to it, and see the details of the job schedule of every query involving this particular table. It will list down all the query jobs that has this table as its source table, and destination table. These query jobs are then grouped by whether they are ad-hoc jobs, live jobs or dead jobs. For each of this job, the counterpart destination table or source table are also noted.
+As seen from the GIF too, for every tables that are involved in the pipeline of the table of interest, you can toggle to it, and see the details of the job schedule of every query involving this particular table. It will list down all pipelines, the ad-hoc, live or dead pipelines that has this table as its source table, and destination table. 
```

Comment:
```suggestion
As seen from the GIF, for every table that is involved in the pipeline of the table of interest, you can select it, and see the details of the job schedule of every query involving this particular table. It will list all ad-hoc, live or dead pipelines that have this table as its source or destination table. 
```